pymysql/connections.py

```diff
@@ -332,7 +335,11 @@ def read_length_encoded_integer(self):
         Length coded numbers can be anywhere from 1 to 9 bytes depending
         on the value of the first byte.
         """
-        c = self.read_uint8()
+        try:
+            c = self.read_uint8()
+        except IndexError:
+            raise _NoMoreColumnsInRow()
+
```

Comment:
This is very performance critical part.
I worry about this `try`~`except` decrease PyMySQL performance lot.
Could you benchmark about SELECT many rows?